src/main/java/org/primefaces/extensions/component/dynaform/DynaFormRenderer.java

```diff
@@ -254,37 +261,37 @@ protected void encodeBody(FacesContext fc, DynaForm dynaForm, List<DynaFormRow>
 					}
 
 					writer.endElement("label");
-				}  else if(element instanceof DynaFormNestedModel){
-					DynaFormNestedModel nestedModel = (DynaFormNestedModel) element;
-					
+				} else if (element instanceof DynaFormModelElement) {
```

Comment:
Can we check for DynaFormControl first and then DynaFormModelElement? In 99% DynaFormControl will be used, so that the else if with DynaFormControl should be better placed before the else if with DynaFormModelElement. Ok?